libfcgi/fcgiapp.c

```diff
@@ -1714,7 +1716,7 @@ static void FillBuffProc(FCGX_Stream *stream)
         memcpy(((char *)(&header)) + headerLen, stream->rdNext, count);
         headerLen += count;
         stream->rdNext += count;
-        if(headerLen < sizeof(header)) {
+        if(headerLen < (int)sizeof(header)) {
```

Comment:
Done the same way with commit 0c7b22b4, although this does not feel right. `sizeof()` returns something of type `size_t`, so the warning occored because headerLen is (signed) _int_ … maybe `headerLen` can be of type `size_t` too? (I don't know, one must review the whole block to answer that.)

Reply:
There is also `ssize_t` which is supposed to represent a signed value of "size" type.

Reply:
From my quick research I learned `ssize_t` is POSIX, so maybe that might not be available on Win32?

Reply:
I'm unfamiliar with the Windows environment. I don't know what it guarantees (or doesn't).

Reply:
Same here. :man_shrugging: